maxim/test_runs/run_utils.py

```
@@ -14,39 +14,45 @@
 async def process_awaitable(awaitable: Awaitable):
     return await awaitable
 
-def get_input_expected_output_and_context_from_row(input_key: Optional[str], expectedOutputKey: Optional[str], contextToEvaluateKey: Optional[str], row: LocalData):
+
+def get_variables_from_row(
+    row: LocalData, data_structure: DataStructure
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)